### ISSUE TITLE
SideNavigation getting vertical and horizontal scroll bars while Collapsing

### ISSUE THREAD

**Issue by gopusrikanth04** [SAP/ui5-webcomponents#2986]
Versions:

    "@ui5/webcomponents": "^1.0.0-rc.12",
    "@ui5/webcomponents-base": "^0.30.0",
    "@ui5/webcomponents-react": "^0.14.5",
    "@ui5/webcomponents-react-base": "^0.14.4",

**Expected behavior**
![side navgation](https://user-images.githubusercontent.com/47174113/112010810-a613f300-8b4d-11eb-8478-c7710006329c.png)

**Screenshots**
![side navigation issue](https://user-images.githubusercontent.com/47174113/112011225-102c9800-8b4e-11eb-9dc6-0ea42ffdeee6.png)

Context

UI5 Web Components version 1.0.0-rc.10
OS/Platform: Win 10
Browser (if relevant): Chrome Version 88.0.4324.192 (Official Build) (x86_64)
Other information: None
Priority:(optional) (Low, Medium, High or Very High)
Very High

Business impact: (mandatory for "Very High")

**Comment by kskondov** [comment]
Hello @SAP/ui5-webcomponents-topic-rd this issue is reproducible in the playground:

https://sap.github.io/ui5-webcomponents/master/playground/components/SideNavigation/

When the side navigation is collapsed a horizontal scrollbar appears bellow the items

**Comment by dimovpetar** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,

Looks like this overflow is caused by a `span` used for accessibility, rendered in `ListItem.hbs`. Can you please check it
````hbs
<span id="{{_id}}-invisibleText" class="ui5-hidden-text">{{_accInfo.listItemAriaLabel}}</span>
````
`display: none;` looks like a solution

Best regards,
Petar

**Comment by plamenivanov91** [comment]
The issue can be tested on the following versions :

"@ui5/webcomponents": "0.0.0-de4752078",
"@ui5/webcomponents-base": "0.0.0-de4752078",
"@ui5/webcomponents-react": "0.14.0-rc.0",
"@ui5/webcomponents-react-base": "0.14.0-rc.0",

It will be officially released with version 1.0.0-rc.15, which will be available in about a month.
